src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableMixedTest.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.sqlserver.jdbc.callablestatement;
+
+import java.sql.*;
+
+import com.microsoft.sqlserver.jdbc.RandomUtil;
+import com.microsoft.sqlserver.jdbc.SQLServerDriver;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+@RunWith(JUnitPlatform.class)
+public class CallableMixedTest {
+
+    @Test
+    public void datatypestest() throws Exception {
+        Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");
+        String connectionString = TestUtils.getConfiguredProperty("mssql_jdbc_test_connection_properties");
+        String tableName = RandomUtil.getIdentifier("TFOO3");
+        String procName = RandomUtil.getIdentifier("SPFOO3");
+
+        try (Connection conn = DriverManager.getConnection(connectionString)) {
+            try (Statement stmt = conn.createStatement()) {
+                try {
+                    stmt.executeUpdate("DROP TABLE " + AbstractSQLGenerator.escapeIdentifier(tableName));
+                    stmt.executeUpdate(" DROP PROCEDURE " + AbstractSQLGenerator.escapeIdentifier(procName));
+                } catch (Exception e) {}

```

Comment:
Exception should not fall through if the test fails to drop these tables. Test should abort with an appropriate error message.